templates/csharp/command-and-response/.{{NewProjectTypeName}}/GettingStarted.md.tpl

```diff
@@ -18,6 +18,11 @@
 6. In the message input field, type and send "helloWorld" to your app to get a response
 {{/enableTestToolByDefault}}
 
+{{^enableTestToolByDefault}}
+## Debug in Test Tool
+Teams App Test Tool allows Teams app developers to test and debug bots in local easily and eliminates the necessity for prerequisites such as Microsoft 365 accounts, develop tunnels, or Teams app and bot registration. See https://github.com/OfficeDev/TeamsFx/wiki/Debug-with-Multiple-Profiles-in-Teams-Toolkit-for-Visual-Studio#start-the-app-in-teams-app-test-tool for more details.
```

Comment:
Better to replace it with aka short link, in case we need to redirect it to learn page.
> See https://github.com/OfficeDev/TeamsFx/wiki/Debug-with-Multiple-Profiles-in-Teams-Toolkit-for-Visual-Studio#start-the-app-in-teams-app-test-tool for more details.